samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -226,6 +230,10 @@ static void radio_config(nrf_radio_mode_t mode, enum transmit_pattern pattern)
 		 */
 		nrf_radio_crc_configure(NRF_RADIO, RADIO_CRCCNF_LEN_Three,
 					NRF_RADIO_CRC_ADDR_SKIP, 0);
+
+		/* preamble, address (BALEN + PREFIX), code indicator, TERM and payload */
+		total_payload_size = 10 + packet_conf.balen + 1 + packet_conf.cilen +
```

Comment:
Please check again those calculations for all PHYs. It looks like sometimes you miss some bytes like length field of the CRC field.

Reply:
Checked and fixed. CRC field is included only for Bluetooth Low Energy Long Range mode. Additionally, the size of the length field was taken into account.